setting-up-tugboat/index.md

```diff
@@ -0,0 +1,510 @@
+# Setting up your Tugboat
+
+- [Connect with your provider](#connect-with-your-provider)
+- [Create a new project](#create-a-new-project)
+- [Add repos to the project](#add-repos-to-the-project)
+- [Select repo settings (optional)](#repository-settings-optional)
+- [Create a Tugboat Config file](#create-a-tugboat-config-file)
+
+> #### Hint::Video Overview
+>
+> If you want to learn more about setting up Tugboat and get a feel for the
+> overall workflow, watch our
+> [Getting Started with Tugboat video](https://www.youtube.com/watch?v=HYTsrm5ORmU)
+> on YouTube.
+
+## Connect with your provider
+
+- [Generic git server](#generic-git-server)
+- [GitHub](#github)
+- [GitLab](#gitlab)
+- [BitBucket](#bitbucket)
+- [Add a link to another git provider](#adding-a-link-to-a-git-provider)
+
+### Generic Git Server
+
+If you're not using GitHub, GitLab, or BitBucket, you can use a generic git
+server with Tugboat. You'll [sign in to Tugboat](https://dashboard.tugboat.qa/)
+using an email address, and when you go to add a repo to your Tugboat project,
+you can link to a Git URL.
+
+> #### Warning::Authenticate with a git provider to use auto-build Preview functionality
+>
+> If your repo isn't connected via Tugboat's GitHub, GitLab, or BitBucket
+> authentication, you won't have the integration features to automatically build
+> Previews from Pull Requests, and other related functionality. If you add
+> GitHub, GitLab, or BitBucket authentication later, you'll need to delete your
+> generic git server from your project, and add it again to use it.
+
+### GitHub
+
+- [How to connect your Tugboat to GitHub](#how-do-i-link-my-tugboat-to-github)
+- [Using the GitHub integration](#using-the-github-integration)
+
+#### How do I link my Tugboat to GitHub?
+
+1. Go to [Sign In](https://dashboard.tugboat.qa/) and select GitHub
+2. Enter your GitHub Username and Password (or Create an Account), complete
+   Two-Factor Authentication
+3. Authorize Tugboat
+
+That's it! Your Tugboat account is now linked to GitHub, and you'll now have
+access to add your repos to your Tugboat project and enjoy all the benefits of
+GitHub integration.
+
+#### Using the GitHub integration
```

Comment:
This is all word-for-word from the UI. Do we really need it in both places?

Reply:
I've included it here for people who are looking at the docs, but haven't actually set up the integration. That information doesn't appear in the UI unless the repo is linked to a git provider, and I wanted users to know what the options are if they haven't done that.